userspace/libsinsp/sinsp.h

```diff
@@ -904,6 +908,8 @@ class SINSP_PUBLIC sinsp : public capture_stats_source
 
 	void set_cri_socket_path(const std::string& path);
 	void set_cri_timeout(int64_t timeout_ms);
+	void set_cri_async(bool async);
```

Comment:
if these don't need to change, they should probably all be construction time params....but that can wait.

Reply:
They're configurable. This only happens at startup but after the objects get constructed. I'll be happy to revisit it later, when we have a decent config system in libsinsp.